sites/friday/src/get-started/laravel/deploy/_index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Deploy Laravel on {{< vendor/name >}}
```

Comment:
ok, this is a bit confusing 
I'm really on "Get Started" but since the title is "Deploy Laravel" I _believe_ I'm on the "Deploy" page under the Get Started section
![image](https://github.com/platformsh/platformsh-docs/assets/2332956/8922804f-df80-447d-9595-0c32a9eff772)